extensions/scarb-cairo-run/src/main.rs

```diff
@@ -43,6 +43,10 @@ struct Args {
     #[arg(long, default_value_t = false)]
     print_full_memory: bool,
 
+    /// Print detailed resources.
+    #[arg(long, default_value_t = false)]
+    print_detailed_resources: bool,
```

Comment:
```suggestion
    print_resource_usage: bool,
```
Just to use the same name, as in cairo-test